client/src/index.jsx

```diff
@@ -1,38 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import $ from 'jquery';
 import Bonuses from './components/bonuses.jsx';
 
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      bundleNumber: 1, //number
-      bonus: null, //object
-      selectedSong: '', //url
-    };
-
-    this.selectSong = this.selectSong.bind(this);
-  }
-  componentDidMount() {
-    $.get(`/bonus/${this.state.bundleNumber}`, (data) => {
-      this.setState({ bonus: data });
-    });
-  }
-
-  selectSong() {
-    console.log('clicked');
-  }
-
-  render() {
-
-    return (
-      <div>
-        <Bonuses bonus={this.state.bonus} selectSong={this.selectSong} />
-      </div>
-    );
-  }
+const App = () => {
```

Comment:
Is it convention to include a container component to your actual service? Seems like you could just render bonuses directly without the containing `<div>`.

Reply:
Ha! I have gotten burned on this before. Sometimes, it wants everything held in a `div`. But you are right. no `div` is okay here. Thanks!